### DISCUSSION TITLE
Calcurators (Furkan Yıldırım)

### DISCUSSION THREAD

**Discussion by Furkanyildirm** [sonsuzus/BilsemPython#58] [Genel]
Grubuma katılmak isteyenler yazabilir.
bilgilerim;
bilsem saatim:  perşembe 3. 4. dersler.
seviyem: ÖYG 2
telefonum: 551 852 56 85
postam: furkanyldmr.2@gmail.com
grup logosu;
![CALCURATORS (1)](https://user-images.githubusercontent.com/114012896/210253066-fe67a62f-473c-42f6-b545-8d2833d70572.jpg)

**Comment by BerraAlkann** [comment]
Merhaba, ben Berra. Öyg-2 Perşembe 5. ve 6. derstenim. Takıma katılabilir miyim?

**Reply by Furkanyildirm** [reply]
evet katılabilirsin telefon numaranı verirsen seni whatsap grubuma katabilirim

**Comment by kaanpinar04** [comment]
Katılabilir miyim?
Kaan Pınar
Salı 3. ve 4. ders
Öyg 2

**Reply by Furkanyildirm** [reply]
Kabul edildin ✅ 
Telefon numaranı yazarsan seni gruba eklerim

**Reply by kaanpinar04** [reply]
0541 106 04 08

**Comment by AdaNilPoyraz** [comment]
Merhaba, ben Ada Nil
Öyg - 2 yim
Salı 3 - 4. derslere geliyorum(16:30 - 18:00 arası)
Katılabilir miyim

**Reply by Furkanyildirm** [reply]
Katılabilirsin . Telefon numaranı yazarsan eklerim seni

**Reply by AdaNilPoyraz** [reply]
Telefon numaram 0536 553 1509

**Comment by Atakanoyg1** [comment]
merhaba katılmak istiyorum adım Atakan ŞAHİN ÖYG 1 Çarşamba 3. ve 4. dersler

**Comment by NecipFG** [comment]
Merhaba ben Necip. ÖYG 1'im. Cumartesi 3-4. derslere giriyorum. 
Eğer takımına beni alırsan çok sevinirim. Şimdiden teşekkür ederim.

Telefon Numaram : 0533 527 77 22
